integration_tests/tests/apply/submit.cy.ts

```diff
@@ -0,0 +1,44 @@
+//  Feature: Referrer submits an application
+//
+//  Scenario: submit an application
+//    Given I am logged in
+//    And I have created an appliation
+//    And I am on the Task List
+//    When I click submit
+//    Then I should see a confirmation page
+
+import TaskListPage from '../../pages/apply/taskListPage'
+import { applicationFactory } from '../../../server/testutils/factories'
+
+context('Applications dashboard', () => {
+  beforeEach(() => {
+    cy.task('reset')
+    cy.task('stubSignIn')
+    cy.task('stubAuthUser')
+
+    // I have created an application
+    cy.fixture('applicationData.json').then(applicationData => {
+      const application = applicationFactory.build({ id: 'abc123', data: applicationData, status: 'inProgress' })
+      cy.wrap(application).as('application')
+    })
+
+    // Given I am logged in
+    cy.signIn()
+  })
+
+  //  Scenario: submit an application
+  // ----------------------------------------------
+  it('shows the dashboard', function test() {
+    cy.task('stubApplicationGet', { application: this.application })
+    cy.task('stubApplicationSubmit', { application: this.application })
+
+    // I visit the Task List Page
+    const page = TaskListPage.visit(this.application)
+
+    // I click submit
+    page.clickSubmit()
+
+    // Then I see a confirmation page
+    cy.get('.govuk-panel__title').should('contain', 'Application complete')
```

Comment:
Is it worth creating the confirmation page and then using the Page.verifyOnPage() method here, so we keep the integration tests clear and high level?

Reply:
good spot, was being lazy :)